downloader/download_test.go

```diff
@@ -103,31 +87,257 @@ func TestDownload(t *testing.T) {
 		},
 	}
 
+	testPrivateRepo = &test{
+		locID: borges.LocationID("2ea758d7c7cbc249acfd6fc4a67f926cae28c10e"),
+		repoIDs: []borges.RepositoryID{
+			borges.RepositoryID("github.com/lwsanty/super-private-privacy-keep-out"),
+		},
+	}
+)
+
+type test struct {
+	locID   borges.LocationID
+	repoIDs []borges.RepositoryID
+}
+
+/* TODO:
+- network errors
+- resolveCommit https://codecov.io/gh/src-d/gitcollector/compare/4dc597c03b4a5106fbf3ba87f35835cf171fb8ee...1310d5b5c391847255f4227eb1746a31871418a3/src/downloader/git.go#L164
+*/
+
+// TestLibraryCreationFailed
+// 1) try to create *siva.NewLibrary using fs with broken OpenFile method
+// <expected> error that contains broken fs mocked error's text
+func TestLibraryCreationFailed(t *testing.T) {
+	h, closer, err := testhelper.NewHelper()
```

Comment:
maybe `close` instead of `closer` would fit better for a function

Reply:
Done